src/hotspot/cpu/aarch64/c1_LIRGenerator_aarch64.cpp

```diff
@@ -426,7 +426,7 @@ void LIRGenerator::do_ArithmeticOp_FPU(ArithmeticOp* x) {
     tmp = new_register(T_DOUBLE);
   }
 
-  arithmetic_op_fpu(x->op(), reg, left.result(), right.result(), NULL);
+  arithmetic_op_fpu(x->op(), reg, left.result(), right.result(), false);
```

Comment:
Ok, my work predates this fixes. Happy it gets fixed upstream.

Reply:
Please note some of the clang warning fixes in this pull request differ from the upstream fixes in JDK-8224851 such as the one mentioned in this file. 

@battleblow It seems like Mikael should sync his changes with JDK-8224851 while we wait for it to be back ported since some of the corrections are functional changes and not just warning suppression. Thoughts?

Upstream changes link:
http://hg.openjdk.java.net/jdk/jdk/rev/e53ec3b362f4